src/main/webapp/favicon.svg

```diff
@@ -0,0 +1,25 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs" width="67.942429mm" height="67.942429mm"><svg xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd" xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape" width="67.942429mm" height="67.942429mm" viewBox="0 0 67.942429 67.942429" version="1.1" id="SvgjsSvg1010" inkscape:version="1.0.1 (3bc2e813f5, 2020-09-07)">
```

Comment:
please remove all inkscape tags, they seem unnecessary

Reply:
this might help: https://superuser.com/questions/739525/removing-personal-data-from-svg-file/739588

Reply:
Is Done.